src/key_grabber.c

```diff
@@ -22,6 +22,7 @@
 #include "key_grabber.h"
 #include "tilda.h"
 #include "xerror.h"
+#include "wizard.h"
```

Comment:
I would like to avoid dependencies on the wizard in other tilda files. Please check if there is a way to move the necessary functions into another file or consider to introduce a new pair of .c/.h files that contain the necessary function.

Reply:
Thanks for the feedback, this has been fixed.